src/scores/categorical/contingency_impl.py

```diff
@@ -1131,10 +1213,15 @@ def make_contingency_manager(
 
 class ThresholdEventOperator(EventOperator):
     """
-    Given a forecast and and an observation, consider an event defined by
-    particular variables meeting a threshold condition (e.g. rainfall above 1mm).
+    See https://scores.readthedocs.io/en/stable/tutorials/Binary_Contingency_Scores.html for
+    a detailed walkthrough showing the use in practice.
```

Comment:
Given we have been moving to relative links where possible, can/should this be a relative path?

Reply:
As above, not sure this is possible